files/image_config/monit/memory_checker

```diff
@@ -111,18 +110,19 @@ def get_running_container_names():
     Returns:
         running_container_names: A list indicates names of running containers.
     """
-    running_container_names = []
-
-    if (is_service_active("docker")):
-        try:
-            docker_client = docker.DockerClient(base_url='unix://var/run/docker.sock')
-            running_container_list = docker_client.containers.list(filters={"status": "running"})
-            running_container_names = [ container.name for container in running_container_list ]
-        except (docker.errors.APIError, docker.errors.DockerException) as err:
-            syslog.syslog(syslog.LOG_ERR,
-                          "Failed to retrieve the running container list from docker daemon! Error message is: '{}'"
-                          .format(err))
-            sys.exit(5)
+    if not is_service_active("docker"):
```

Comment:
@liorghub there is no point of having this check in `get_running_container_names` function. Maybe move it to out to the script root? See my previous comment ...

Reply:
Done.